integration/presets/longRunningApps.ts

```diff
@@ -0,0 +1,47 @@
+import type { LongRunningApplication } from '../models/longRunningApplication';
+import { longRunningApplication } from '../models/longRunningApplication';
+import { envs } from './envs';
+import { next } from './next';
+import { react } from './react';
+import { remix } from './remix';
+
+/**
+ * A list of long-running applications that can be used in tests.
+ * These are applications that are started once and then used for all tests,
+ * making the tests run faster as the app doesn't need to be started for each test.
+ */
+export const createLongRunningApps = () => {
+  const configs = [
+    { id: 'react.vite.withEmailCodes', config: react.vite, env: envs.withEmailCodes },
```

Comment:
❓ Is this `id` just a string, or is it matching something?

Reply:
Just a string identifier you can use to boot a group of apps or a specific app, eg:

```
E2E_APP_ID=react.vite.* npm run test:integration:base -- --grep @generic
```